packages/generate/test/params.test.ts

```diff
@@ -274,7 +274,7 @@ SELECT (SELECT __param__test)`
       tuple: args,
     });
 
-    assert.deepEqual(Object.values(complexResult.tuple), Object.values(args));
+    assert.deepEqual(complexResult.tuple, args);
```

Comment:
This changed for no good reason, but our macos tests were failing with a test timeout until I did this update. 🫠